Dockerfile

```diff
@@ -8,5 +8,6 @@ COPY cogeo_mosaic_tiler/ cogeo_mosaic_tiler/
 COPY setup.py setup.py
 
 # Install dependencies
+RUN pip install cython==0.28 --no-cache-dir
```

Comment:
Not sure if this is needed because of https://github.com/lambgeo/geo-layer/blob/master/Dockerfile#L8

Reply:
It failed when I tried to build it without including the line here.

But if `rio-tiler-mvt` is an optional import, that can definitely be removed.

Reply:
Oh we should use `-geolayer` so it has most of the heavy libs 


```
gdal2.4-py3.7-geolayer: Pulling from lambgeo/lambda
...
Digest: sha256:d5dcfddb29bb638087dcb607ba08c700f64545bb5a40c910c3b49e22d6ecf66c
Status: Downloaded newer image for lambgeo/lambda:gdal2.4-py3.7-geolayer
bash-4.2# pip install rio-tiler-mvt
...
Successfully built rio-tiler-mvt vtzero
Installing collected packages: vtzero, rio-tiler-mvt
Successfully installed rio-tiler-mvt-0.0.1.dev1 vtzero-0.0.1b1
..
bash-4.2# python
Python 3.7.7 (default, Mar 17 2020, 13:17:02
Type "help", "copyright", "credits" or "license" for more information.
>>> import cython
>>> cython.__version__
'0.28'
>>> exit()

Reply:
Ok cool, I think I updated to the right docker image?